_includes/integrations/webcomponent-tech-ref.md

```diff
@@ -0,0 +1,590 @@
+
+To use a cloud version of Web Component, the integration is accessable from [JSDelivr](https://www.jsdelivr.com/package/npm/@tinymce/tinymce-webcomponent).
+
+For example:
+
+```html
+<script src="{{site.webcomponent_url}}"></script>
+```
+
+### Installing the TinyMCE Web Component integration
+
+To add the {{site.productname}} Web Component integration to a JavaScript project, on command line or command prompt, run the following:
+
+- NPM users:
+
+    ```sh
+    npm i --save @tinymce/tinymce-webcomponent
+    ```
+
+- Yarn users:
+
+    ```sh
+    yarn add @tinymce/tinymce-webcomponent
+    ```
+
+### Loading TinyMCE
+
+The integration requires both the {{site.productname}} Web Component (`tinymce-webcomponent.min.js`) and {{site.productname}} (`tinymce.min.js`).
+
+If {{site.productname}} is not available for {{site.productname}} Web Component, the latest version of {{site.productname}} will be automatically loaded from the {{site.cloudname}}.
+
+To use {{site.productname}} Web Component with a self-hosted copy of {{site.productname}}, ensure that the self-hosted copy of {{site.productname}} can be loaded in the same location as the {{site.productname}} Web Component (such as the same web page).
+
+To use {{site.productname}} from the {{site.cloudname}}, add the `api-key` attribute to the `tinymce-editor` element with an API from [{{site.accountpage}}]({{site.accountpageurl}}).
+
+### Configuring the editor
+
+The editor is configured by setting attributes on the `tinymce-editor` custom element.
+All attributes may be represented as strings, however some attributes will accept boolean or number inputs. Where an array, object, or function is a valid input; examples have been provided.
+
+#### Setting the initial content
+
+To set the initial editor content, add encoded HTML to the `tinymce-editor` element as content, such as:
+
+```html
+<tinymce-editor>
+  &lt;p&gt;This will be the initial content of the editor.&lt;/p&gt;
+</tinymce-editor>
+```
+
+The editor will load the text content in the `tinymce-editor` element as the initial HTML.
+
+> **Important**: The HTML must be encoded as text to prevent injected javascript from running, which may have an unintended impact on the surrounding document.
+
+#### Loading plugins
+
+To add plugins and extend the functionality of the editor, add a string of plugin identifiers using the `plugins` attribute, such as:
+
+```html
+<tinymce-editor plugins="advlist autolink link image lists charmap print preview"></tinymce-editor>
+```
+
+For a list of available plugins, see: [Add plugins to {{site.productname}}]({{site.baseurl}}/plugins/).
+
+#### Setting the editor width
+
+To set the width of the editor (content area and user interface), use the `width` attribute. For pixel-based widths, provide a number; otherwise provide a string. For example:
+
+##### Setting a pixel-based editor width
+
+```html
+<tinymce-editor width=300></tinymce-editor>
```

Comment:
Attributes should always be quoted. It is always a string, but if the string only contains numbers then it will be parsed into a number by TinyMCE.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1659/commits/ff98e3ed3479f619921990f2b28efb25ab1d199d